src/redux/rocketReducer/RocketReducer.js

```diff
@@ -0,0 +1,27 @@
+import { RocketConstants } from './RocketAction';
+
+const initialState = {
+  rockets: [],
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const ReducerRocket = (state = initialState, { type, payload }) => {
+  if (type === RocketConstants.FETCH_ROCKET) {
+    return {
+      rockets: [...state.rockets, ...payload],
+    };
+  }
+  if (type === RocketConstants.RESERVE_ROCKET) {
+    const newState = state.rockets.map((rocket) => {
+      if (rocket.id !== payload) { return rocket; }
+      if (rocket.id === payload && rocket.reserved === true) {
+        return { ...rocket, reserved: false };
+      }
+      return { ...rocket, reserved: true };
+    });
+    return {
+      rockets: [...newState],
+    };
+  }
+  return state;
+};
```

Comment:
- [ ] I see you separate reducers and actions. Like the last project, you can use [duck pattern.](https://medium.com/swlh/the-good-the-bad-of-react-redux-and-why-ducks-might-be-the-solution-1567d5bdc698)